### DISCUSSION TITLE
Ipads removed from baseband list

### DISCUSSION THREAD

**Discussion by p2houd** [LukeZGD/Legacy-iOS-Kit#587] [General]
[448716](https://github.com/LukeZGD/Legacy-iOS-Kit/commit/a9f136a5792b871f201123ae561605667e1fae60)
Can I use previous commit to downgrade my iPad3,3 without jailbreaking and custom ipsw?
Why ipads are removed from baseband list?

**Comment by LukeZGD** [comment]
No. For the iPad3,3 (and all iPad 2 and 3 cellular devices), custom IPSWs will ALWAYS be used to disable baseband update.

Also, that commit is old and irrelevant now, since baseband update has been re-enabled on the iPads that actually need it. For 32-bit iPads, baseband update will be done on the following only: iPad mini, and iPad 4.

**Reply by p2houd** [reply]
Does that mean I need to jailbreak iPad3,3 first to flash this custom ipsw?
It asks for an ssh connection via root after custom ipsw generated and I disabled jailbreak.

**Reply by LukeZGD** [reply]
you always need to jailbreak to be able to restore/downgrade anyway. for kdfu mode

**Reply by p2houd** [reply]
For me no need jailbreak for iPhone 5 and downgraded succsessfully with SHSH.
Jailbreak you mean only before downgrade? Once downgraded, jailbreak is removed if i disable it, right?

**Reply by LukeZGD** [reply]
> For me no need jailbreak for iPhone 5 and downgraded succsessfully with SHSH.

iphone 5 is an a6 device, so you were able to do that easily especially with a mac or ios device with ipwnder. for a5 devices, things are a bit different. unless you have an arduino or pi pico for checkm8-a5, you will need to do the jailbreak procedure for kdfu mode.

> Jailbreak you mean only before downgrade? Once downgraded, jailbreak is removed if i disable it, right?

yes and yes
